### PR TITLE
[MIG] [12.0] mgmtsystem manual

### PR DESCRIPTION
**Pull request by gurneyalex**
depends on :
 - [x] #244

### CODE REVIEW

**Review by i-vyshnevska — APPROVED** [review]
LG

File: mgmtsystem_manual/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Management System - Manual",
-    "version": "11.0.1.0.0",
+    "version": "12.0.1.0.0",
     "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
     "website": "http://www.savoirfairelinux.com",
```
Comment (simahawk) [inline mgmtsystem_manual/__manifest__.py]: https://github.com/OCA/management-system

File: mgmtsystem_manual/models/mgmtsystem_manual.py [hunk 2]
```diff
@@ -1,10 +1,15 @@
+# -*- coding: utf-8 -*-
```
Comment (simahawk) [inline mgmtsystem_manual/models/mgmtsystem_manual.py]: remove encoding

File: mgmtsystem_manual/readme/ROADMAP.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+No known issues
```
Comment (simahawk) [inline mgmtsystem_manual/readme/ROADMAP.rst]: you can remove this

File: mgmtsystem_manual/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    "depends": [
+        'document_page',
+        'mgmtsystem',
+    ],
+    "data": [
+        'data/mgmtsystem_manual.xml',
+        'views/mgmtsystem_manual.xml',
+        'views/document_page.xml',
+    ],
+    "demo": [],
```
Comment (simahawk) [inline mgmtsystem_manual/__manifest__.py]: remove demo

File: mgmtsystem_manual/models/mgmtsystem_manual.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+# Copyright 2019 Odoo Community Association
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class MgmtSystemManual(models.Model):
+    _inherit = 'mgmtsystem.system'
+    _description = 'Manual'
```
Comment (simahawk) [inline mgmtsystem_manual/models/mgmtsystem_manual.py]: _description is not needed since the model already exists

**Review by simahawk — APPROVED** [review]
minor glitches, overall LG

**Pull request merged**